### PR TITLE
Updating template so that Xcode 7 doesn't ask to convert swift to latest syntax

### PR DESCRIPTION
**Pull request by wmathurin**
@rwhitley pointed out that Xcode 7 was offering to convert swift syntax the first time app generated with forceios --apptype=native_swift were opened.

### CODE REVIEW

File: build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -80,7 +80,7 @@
 		4FD6C4A11B755265002F9F90 /* Classes */ = {
 			isa = PBXGroup;
 			children = (
-				4FD6C49A1B755242002F9F90 /* __NativeSwiftTemplateAppName__-Bridging-Header.h */,
```
Comment (wmathurin) [inline build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__.xcodeproj/project.pbxproj]: Unrelated

**Comment by bhariharan** [comment]
:+1:
